build.gradle

```diff
@@ -93,11 +93,9 @@ allprojects {
     }
 }
 
-group 'org.apache.cassandra'
+group 'org.apache.cassandra.sidecar'
```

Comment:
to avoid having to declare the group whenever a new subproject is added. Avoiding the issues encountered in CASSSIDECAR-190